src/components/Text.tsx

```diff
@@ -119,8 +133,11 @@ export default function Text({
 		<ink-text
 			style={{flexGrow: 0, flexShrink: 1, flexDirection: 'row', textWrap: wrap}}
 			internal_transform={transform}
+			internalAccessiblity={{
```

Comment:
nit: name internalAccessibility doesn't seem consistent with other names. Should either be internal_accessibility
or consider switching to just surfacing 
ariaHidden directly.